kratos/solving_strategies/convergencecriterias/mixed_generic_criteria.h

```diff
@@ -406,17 +406,17 @@ class MixedGenericCriteria : public ConvergenceCriteria< TSparseSpace, TDenseSpa
 
     /**
      * @brief Finds the var local key in the mLocalKeyMap for 
-     * a gifen dof. If the variable does not exist in mLocalKeyMap
+     * a gifen DOF. If the variable does not exist in mLocalKeyMap
      * this function returns false
-     * @param iDof the dof
+     * @param itDof the DOF iterator
      * @param rVarLocalKey variable local key 
      * @return dof variable is found or not
      */
     bool FindVarLocalKey(
-        typename DofsArrayType::const_iterator iDof,
+        typename DofsArrayType::const_iterator itDof,
```

Comment:
how about passing the variable? The dof is only used to access the variable

Reply:
well I used the dof to minimize the duplicated code in the trilinos side, what is the advantage on passing the variable instead?